styles.css

```diff
@@ -0,0 +1,137 @@
+:root {
```

Comment:
```
Suggestion: you can use a separate file for variables. The rest looks good.
```

Reply:
Will add variable css file later, while file restructuring. Thank you for suggestion